# object-colon-spacing
冒号后面需要有空格

A space is required after the colon

## Rule Details

该规则旨在确保代码格式统一，从而使代码更加可读。

This rule aims to ensure that the code is formatted uniformly, making the code more readable.

错误示范：

Examples of **incorrect** code for this rule:

```js

const person = {
    name:'Vinsea',
    github : 'Vinsea',
    key :'Eslint',
}

```
正确示范：

Examples of **correct** code for this rule:

```js

const person = {
    name: 'Vinsea',
    github: 'Vinsea',
    key: 'Eslint',
}

```

### Options
none

